### ISSUE TITLE
Render formatting in issue description

### ISSUE THREAD

**Issue by jneidel** [mk-5/fjira#55]
**Is your feature request related to a problem? Please describe.**

Formatting like bolding, italic, lists with sub items, headlines is just printed as the text without being applied.

<img width="664" alt="Screenshot 2023-08-28 at 15 43 38" src="https://github.com/mk-5/fjira/assets/25589715/13530e28-7fc6-4a68-94ef-127ed42b3a80">

**Describe the solution you'd like**

I would like the text to be formatted as such.
In my example it would be:

List:
```
* Add to /profiles
** profile picture

- Add to /profiles
  - profile picture
```

Bolding:
`*Technical Refinement*` -> **Technical Refinement**

Headlines:
`h2. ACs` -> `ACs` in yellow or with background color (see [glow](https://github.com/charmbracelet/glow) for inspiration)

**Comment by mk-5** [comment]
that will be tricky to implement - but it will be really nice to have it

**Comment by mk-5** [comment]
i did some research - it looks that there is no markdown parser for tcell. I found such a note in tcell docs:

![image](https://github.com/mk-5/fjira/assets/31850900/9c3d5a2f-340a-4b19-a22b-b4ebe02c6c66)

I agree, implementing the whole markdown stuff using tcell is not super trivial. Implementing some basics markdown like bold, punctuations, italic - should be easy to implement. Maybe that's a good compromise

**Comment by jneidel** [comment]
Yeah, no need to implement the whole spec, just the most common usages :smile:
